Makefile

```diff
@@ -87,6 +87,10 @@ KUBEADM_BOOTSTRAP_CONTROLLER_IMG ?= $(REGISTRY)/$(KUBEADM_BOOTSTRAP_IMAGE_NAME)
 KUBEADM_CONTROL_PLANE_IMAGE_NAME ?= kubeadm-control-plane-controller
 KUBEADM_CONTROL_PLANE_CONTROLLER_IMG ?= $(REGISTRY)/$(KUBEADM_CONTROL_PLANE_IMAGE_NAME)
 
+# operator
+OPERATOR_IMAGE_NAME ?= operator-controller
```

Comment:
`operator-controller`, uhmm ...
what about `cluster-api-operator`? this IMO will makes a nice pair with `cluster-api-controller`